x-pack/plugins/ingest_pipelines/public/application/components/pipeline_editor/components/processor_form/processors/inference.tsx

```diff
@@ -28,42 +28,32 @@ const { emptyField, isJsonField } = fieldValidators;
 
 const INFERENCE_CONFIG_DOCS = {
   regression: {
-    path: 'inference-processor.html#inference-processor-regression-opt',
     linkLabel: i18n.translate(
       'xpack.ingestPipelines.pipelineEditor.inferenceForm.inferenceConfigField.regressionLinkLabel',
       { defaultMessage: 'regression' }
     ),
   },
   classification: {
-    path: 'inference-processor.html#inference-processor-classification-opt',
     linkLabel: i18n.translate(
       'xpack.ingestPipelines.pipelineEditor.inferenceForm.inferenceConfigField.classificationLinkLabel',
       { defaultMessage: 'classification' }
     ),
   },
 };
 
-const getInferenceConfigHelpText = (esDocsBasePath: string): React.ReactNode => {
+const getInferenceConfigHelpText = (esDocsLink1: string, esDocsLink2: string): React.ReactNode => {
```

Comment:
Can we name these `regressionDocsLink` and `classificationDocsLink` respectively?